### PR TITLE
Improve assertions to ensure well-formed img elements

### LINKED ISSUE #1021: Tests pass before finished typing
Challenge http://www.freecodecamp.org/challenges/waypoint-mobile-responsive-images has an issue. Please describe how to reproduce it, and include links to screen shots if possible.

The following will pass the test:
```html
<img class="img-responsive" src="http://bit.ly/fcc-kittens2"
```

Note: it's missing a `>` at the end of it

Im thinking this is a flaw with it rendering it on the fly creating the dom elements even if it's not complete? But it leads to a false sense of being done when you really aren't done with the element to pass it.  Subjective I guess depending on your view of it?

### PR DESCRIPTION
**Pull request by ptsurbeleu**
Reused the original approach from Waypoint: Headline with the h2 Element (http://freecodecamp.com/challenges/waypoint-headline-with-the-h2-element) by using the very same technique. Also, slightly adjusted assertion messages to use similar wording (for consistency).

Here is the snip of how these changes would react upon not-well-formed img element:
image

And here is the snip of how these changes would react upon the completed challenge:
image

Appreciate your feedback very much!

### CODE REVIEW

**Comment by ahstro** [comment]
Thank you for this, but you'll need to read [Pull Request Contribute](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Pull-Request-Contribute) - specifically the parts regarding not putting issue numbers in the title and instead putting them in the description - and edit hte pull request accordingly before we can merge it.

File: seed/challenges/bootstrap.json [hunk 1]
```diff
@@ -96,9 +96,10 @@
         "Fortunately, with Bootstrap, all we need to do is add the <code>img-responsive</code> class to your image. Do this, and the image should perfectly fit the width of your page."
       ],
       "tests": [
-        "assert($(\"img\").length > 1, 'You should have a total of two images.')",
+        "assert($(\"img\").length === 2, 'You should have a total of two images.')",
+        "assert(editor.match(/<img/g) && editor.match(/<img.*>/g).length === 2 && editor.match(/<img/g).length === 2, 'Make sure your new <code>img</code> element has a closing angle bracket.')",
         "assert($(\"img\").hasClass(\"img-responsive\"), 'Your new image should have the class <code>img-responsive</code>.')",
-        "assert(new RegExp(\"http://bit.ly/fcc-running-cats\", \"gi\").test($(\"img.img-responsive\").attr(\"src\")), 'Add a second image with the <code>src</code> of <code>http&#58;//bit.ly/fcc-running-cats</code>.')"
+        "assert(new RegExp(\"http://bit.ly/fcc-running-cats\", \"gi\").test($(\"img.img-responsive\").attr(\"src\")), 'Your new image should have <code>src</code> attribute of <code>http&#58;//bit.ly/fcc-running-cats</code>.')"
```
Comment (ahstro) [inline seed/challenges/bootstrap.json]: Why is this change here? It seems unrelated to the issue being fixed. (Also, you need to put `a` between `have` and `src` for it to be grammatically correct.)

Comment (ptsurbeleu) [inline seed/challenges/bootstrap.json]: This was an extra change to adjust wording of the assertion to be consistent with the rest of the messages, but I agree with you it should be an isolated commit. I will split these into two separate commits and incorporate your feedback about correctness.

**Comment by ptsurbeleu** [comment]
Finally, after reading PR guidelines for at least 10 times I understand what does "Do NOT add issue numbers to the title" mean. No issue numbers in the pull request title!!

Not sure if it's only me being confused by the guidelines but it feels like there is a better way to phrase DON'Ts to avoid confusion.

**Comment by BerkeleyTrue** [comment]
@ptsurbeleu WIki is freely editable and I encourage it as long as the end result is the same.
